### PR TITLE
Remove LESS

### PR DESCRIPTION
**Pull request by mikewheaton**
This should get us all the way to completing #56. @battletoilet, please have a look at the build process to see if I've missed anything.

### CODE REVIEW

**Comment by Jahnp** [comment]
Have you checked out the copy/build steps in FabricBuild.js?

**Comment by mikewheaton** [comment]
Where is FabricBuild.js?

File: gulp/ComponentSamples.js [hunk 1]
```diff
@@ -69,21 +69,6 @@ gulp.task('ComponentSamples-moveJS', function() {
             .pipe(gulp.dest(Config.paths.distSamples + '/Components'));
 });
 
-// Style Linting
-// ----------------------------------------------------------------------------
-
-gulp.task('ComponentSamples-styleHinting',  function() {
```
Comment (Jahnp) [inline gulp/ComponentSamples.js]: Interesting--is there no linting for SASS built in that we can switch to? We definitely want that included, very soon if not right now in this PR. I realize that's scope creep but it's an important aspect of development we shouldn't lose.

File: gulp/modules/Config.js [hunk 2]
```diff
@@ -57,8 +54,7 @@ var Config = function() {
 		{src: this.paths.distComponents, dest: "/content/components/"},
 		{src: this.paths.distCSS, dest: "/content/css/"},
 		{src: this.paths.distJS, dest: "/content/scripts/"},
-		{src: this.paths.distLess, dest: "/content/less/"},
-        {src: this.paths.distSass, dest: "/content/sass/"}
+    {src: this.paths.distSass, dest: "/content/sass/"}
```
Comment (Jahnp) [inline gulp/modules/Config.js]: Nitpick, but mind the indentation--looks like a spaces/tabs diff issue

File: gulp/modules/ErrorHandling.js [hunk 3]
```diff
@@ -186,54 +184,7 @@ var ErrorHandling = function() {
         });
     };
     /**
-     * LESSHint error handler
```
Comment (Jahnp) [inline gulp/modules/ErrorHandling.js]: Same as above: even if we aren't adding SASS linting here, it's super important that we do add it shortly.

Comment (battletoilet) [inline gulp/modules/ErrorHandling.js]: Adding it back right now

**Comment by Jahnp** [comment]
Overall, looks really good. Biggest issue I can see is a lack of SASS linting/error reporting now that LESS is no longer part of the build at all. 

Also, this probably doesn't need to be addressed in this PR, we should consider whether we need some of the leftover config bits from supporting 2 preprocessors, e.g. running a task for setting SassMode explicitly (it's the only mode now) and the sassExtension prop in Config.js.

**Comment by Jahnp** [comment]
Also, please remove component-manifest-template.less--that's no longer needed either.
